lib/tests/mocks/badTextFormatted.js

```diff
@@ -7,6 +7,22 @@ const badTextFormatted = `
 02:00:10,817 --> 02:00:16,818
 {an 1}
 
+3
+03:00:10,817 --> 03:00:16,818
+>>Hello
+
+4
+04:00:10,817 --> 04:00:16,818
+World 
```

Comment:
Can you add `<br>` to the front of one of these entries so we know it is being tested?